packages/lib/services/DecryptionWorker.ts

```diff
@@ -246,7 +287,20 @@ export default class DecryptionWorker {
 							throw error;
 						}
 
-						if (options.errorHandler === 'log') {
+						if (options.errorHandler === 'log' || options.errorHandler === 'dispatchAndLog') {
+							const errorType = Object.getPrototypeOf(error);
```

Comment:
While keeping track of error prototypes works (then later checking whether decryption errors  have a specific prototype) works in this case (actually, **TODO(@personalizedrefrigerator)** double-check this) and _is_ a way of keeping track of error types (and not the individual errors themselves), it's probably better to keep track of `error.code` values.

Not all errors have an `error.code`, however...

Reply:
This has been changed to use `error.code` or, if there is no code, a string version of `error`.